src/linux/cio_linux_server_socket.c

```diff
@@ -71,14 +71,15 @@ static void accept_callback(void *context)
 	} else {
 		struct cio_socket *s = ss->alloc_client();
 		if (cio_likely(s != NULL)) {
-			enum cio_error err = cio_linux_socket_init(s, client_fd, ss->loop, ss->free_client);
+			enum cio_error err = cio_linux_socket_init(s, client_fd, ss->impl.loop, ss->free_client);
 			if (cio_likely(err == CIO_SUCCESS)) {
 				ss->handler(ss, ss->handler_context, err, s);
 			} else {
 				close(client_fd);
 				ss->free_client(s);
 			}
 		} else {
+			// TODO: call handler with error
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS1135)